app/inbound/get-payment-request-by-invoice-number.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../data')
+
+const getPayementRequestbyInvoiceNumber = async (settlement, transaction) => {
```

Comment:
`getPay**e**mentRequestbyInvoiceNumber`

Reply:
ooops thought we were doing ye olde payements